SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -483,6 +489,21 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 		return Region->GetDisplayNameTextByValue(static_cast<int64>(PrimaryDeploymentRegionCode.GetValue()));
 	}
 
+	void SetMainDeploymentCluster(const FString& NewCluster);
+	FORCEINLINE FString GetRawMainDeploymentCluster() const
```

Comment:
Why does this one need a raw version and the sim one not?

Reply:
there is a sim player version: GetRawSimulatedPlayerCluster

Reply:
There is GetRawSimulatedPlayerCluster in this file

Reply:
Oh, I see it now - I think GitHub was having a moment again.